src/app/about/OfficerCard.module.css

```diff
@@ -0,0 +1,60 @@
+.officerCard {
+  /* align */
+  width: 250px;
+  max-width: 100%;
+  height: 320px;
+  /* style */
+  background-color: rgba(0, 0, 0, 0.4);
+  border-radius: 10px;
+  /* content */
+  padding: 1rem;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.portrait {
+  /* align */
+  height: 180px;
+  width: 180px;
+  /* style */
+  object-fit: cover;
+  border-radius: 5px;
+}
+
+.officerCard h3 {
+  /* style */
+  font-size: 1.3rem;
+  font-weight: 700;
+  text-align: center;
+}
+
+.officerCard h4 {
+  /* style */
+  font-size: 1.25rem;
+  font-weight: 700;
+  text-align: center;
+}
+
+.socialLinks {
+  /* content */
+  display: flex;
+  gap: 1rem;
+}
+
+/* 
+The website logo is slightly smaller than the other logos.
+This scales it up a tiny bit.
+This is where the logo comes from:
+https://fonts.google.com/icons?selected=Material%20Symbols%20Outlined%3Alanguage%3AFILL%401%3Bwght%40400%3BGRAD%400%3Bopsz%4024

```

Comment:
[Comment]: Good callout & use of comments. May be worth looking into later, but let's accept this limitation for now. I'll sync with you later and see if we should open a backlog ticket to dig into this more.